catalog/app/components/JsonEditor/JsonEditor.js

```diff
@@ -70,27 +66,25 @@ function JsonEditor({
     [changeValue, onChange],
   )
 
+  const columnData = R.last(columns)
+  const columnPath = R.slice(0, columns.length - 1, fieldPath)
```

Comment:
fyi `R.slice(0, x)` is equivalent to `R.take(x)`

Reply:
Thanks, I forget about it